test/asana/helpers/test_extract_task_fields_from_pull_request.py

```diff
@@ -44,9 +44,6 @@ def test_html_body(self):
             '<A href="https://foo.bar/baz">https://foo.bar/baz</A>',
             "✍",
             "TEST_USER_ASANA_DOMAIN_USER_ID",
-            "<strong>",
-            "Description:",
```

Comment:
Why was "Description" removed?

Reply:
It felt redundant once Summary was bolded, but I don't feel strongly about that at all

Reply:
I think "Summary" is just in our CODEZ `git pr` template, but SGTM is used for other repos (and is open sourced now!) so we shouldn't necessarily optimize for the `git pr` template

Reply:
Ah makes sense, I'll add it back